### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by mayankrajendrat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Sujit-Banne) [inline index.html:6]: Link style.css and style.js

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <title>Document</title>
+</head>
+<body>
```
Comment (SwamyBurande55MERN) [inline index.html:10]: link css file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (codebyAvijit) [inline index.html]: hey mayank can you please change the document title

File: index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+</head>
+<body>
+
```
Comment (BhargaviMadishetty) [inline index.html:11]: Hi Mayank, Link the Script file

File: index.html (hunk 1 above)
Comment (mayankrajendrat) [reply index.html:6]: Added Styles and Script Refrences

File: index.html (hunk 2 above)
Comment (mayankrajendrat) [reply index.html:10]: Added Styles and Script References

**Pull request merged**